ONSdigital/design-system
# Don't render list if only one item

### What is the context of this PR?
Fixes: #1462 

### How to review
- Check lists with more than one item still work in the same way
- Check lists with only one item now just render in `<p>` tags and not a list with no list styling added
